query/common/src/main/java/org/codice/ddf/admin/common/poller/BaseEnumValuePoller.java

```diff
@@ -18,13 +18,17 @@
 
 public abstract class BaseEnumValuePoller<V, T> implements EnumValuePoller<V, T> {
 
+    private static final String BINDING_ENUM_VALUE = "Binding enum value for class ";
```

Comment:
✏️ You could consider using format strings instead. So this first one would be something like `Binding enum value for class %s` and then you'd call `String.format` instead of concatenating when you invoke the `updateGraphQLSchema` method. Gives you a bit more control over the formatting.